### PR TITLE
Cleaning + adding new example

### PR DESCRIPTION
**Pull request by thibaultduponchelle**
A new example if we only want to get the velocity without report.

### CODE REVIEW

File: examples/print-repository-velocity.pl [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+use GitHub::MergeVelocity;
+
+my $mv = GitHub::MergeVelocity->new(
+    url => [
+        'neilbowers/PAUSE-Permissions',
+        'https://github.com/oalders/html-restrict/issues',
+    ]
+);
+
+my @repos = map { $mv->_repository_for_url($_) } $mv->_report_urls;
```
Comment (oalders) [inline examples/print-repository-velocity.pl:13]: We should probably make `_repository_for_url` and `_report_urls` public if we're going to do this. I try to make the majority of the API private by default unless there's a reason for it to be public. Doing the above demonstrates that those methods should probably be public.

**Review by oalders — CHANGES_REQUESTED** [review]
Great idea to add this example. Just had one suggestion.

**Comment by oalders** [comment]
@thibaultduponchelle did you intend to close this?

**Comment by thibaultduponchelle** [comment]
Hi @oalders 

Hmm I thought deleting the fork would not close the PR... Actually it's not that bad as I'm probably not going to work on the remaining changes anytime soon.

**Pull request closed without merging**